webapp/channels/src/components/file_attachment/file_thumbnail/file_thumbnail.tsx

```diff
@@ -19,47 +19,49 @@ type Props = {
     fileInfo: FileInfo;
 }
 
-export default class FileThumbnail extends React.PureComponent<Props> {
-    render() {
-        const {fileInfo} = this.props;
-        const type = getFileType(fileInfo.extension);
+const FileThumbnail = ({
+    fileInfo,
+    enableSVGs,
+}: Props) => {
+    const type = getFileType(fileInfo.extension);
 
-        let thumbnail;
-        if (type === FileTypes.IMAGE) {
-            let className = 'post-image';
+    let thumbnail;
```

Comment:
Since we are returning at the end the thumbnail, I think this can be simplified changing the pattern from:
```
let thumbnail;
if (...) {
    thumbnail = X;
} else if (...) {
    thumbnail = Y;
} else {
    thumbnail = Z;
}
return thumbnail;
```
to:
```
if (...) {
    return X
}
if (...) {
    return Y;
}

return Z;
```